editor/src/clj/editor/code/script.clj

```diff
@@ -472,7 +472,7 @@
   ;; We then strip go.property() declarations and recompile if needed.
   (let [lines (:lines user-data)
         proj-path (:proj-path user-data)
-        bytecode-or-error (script->bytecode lines proj-path :32-bit)]
+        bytecode-or-error (script->bytecode lines proj-path :64-bit)]
```

Comment:
We don't actually use the `bytecode` result. We're only interested in the syntax errors.